contrib/go.mongodb.org/mongo-driver.v2/mongo/mongo.go

```diff
@@ -56,6 +54,13 @@ func (m *monitor) Started(ctx context.Context, evt *event.CommandStartedEvent) {
 		tracer.Tag(ext.SpanKind, ext.SpanKindClient),
 		tracer.Tag(ext.DBSystem, ext.DBSystemMongoDB),
 	}
+	if m.cfg.maxQuerySize != 0 {
+		b, _ := bson.MarshalExtJSON(evt.Command, false, false)
+		if m.cfg.maxQuerySize > 0 && len(b) > m.cfg.maxQuerySize {
+			b = b[:m.cfg.maxQuerySize]
+		}
+		opts = append(opts, tracer.Tag(m.cfg.spanName, string(b)))
+	}
```

Comment:
@colinking I'm wondering if `m.cfg.maxQuerySize == 0` should just default to the default behaviour. @rarguelloF What do you think?